src/js/drop.js

```diff
@@ -124,6 +124,23 @@ function createContext(options={}) {
         throw new Error('Drop Error: You must provide a target.');
       }
 
+      var dataPrefix = 'data-' + drop.classPrefix;
```

Comment:
Mind changing this and the variables below to use ES6 string interpolation? e.g.
```javascript
const dataPrefix = `data-${drop.classPrefix}`;
```